netskope_connector.py

```diff
@@ -1976,7 +1976,7 @@ def _update_file_helper(self, action_result):
                 return action_result.get_status()
             action_result.add_data({})
             summary = action_result.update_summary({})
-            summary['total_files'] = action_result.get_data_size()
```

Comment:
Do we need to remove this field? 
This can break the playbooks of other customers who are using this field in their playbooks.

Reply:
The 'total_files' summary key was unnecessary and did not provide any useful information to the users. We have replaced it with an equivalent summary key of 'total_hashes' which will display the total entities updated on Netskope UI. 
For backward compatibility, we have added this point in the readme.

Reply:
I see. Thanks!